content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,103 @@
+
+Today we introduce the R package `epidatr`, available [on CRAN](https://cloud.r-project.org/web/packages/epidatr/index.html), with the source and development [on github](https://github.com/cmu-delphi/epidatr).
+
+This package is designed to streamline the downloading and usage of data from the Delphi Epidata API. It provides a simple R interface to the API, including functions for downloading data, parsing the results, and converting the data into a tidy format. The API stores a historical record of all data, including corrections and updates, which is particularly useful for accurately backtesting forecasting models. We also provide packages for downstream data processing ([epiprocess](https://github.com/cmu-delphi/epiprocess)) and modeling ([epipredict](https://github.com/cmu-delphi/epipredict)).
+
+## Usage
+
+```
+library(epidatr)
+# Obtain the smoothed covid-like illness (CLI) signal from the Facebook survey as it was on April 10, 2021 for the US
+epidata <- pub_covidcast(
+    source = "fb-survey",
+    signals = "smoothed_cli",
+    geo_type = "nation",
+    time_type = "day",
+    geo_values = "us",
+    time_values = epirange(20210101, 20210601),
+    as_of = 20210601
+)
+epidata
+```
+
+```
+# A tibble: 151 × 15
+   geo_value signal       source geo_type time_type time_value
+   <chr>     <chr>        <chr>  <fct>    <fct>     <date>
+ 1 us        smoothed_cli fb-su… nation   day       2021-01-01
+ 2 us        smoothed_cli fb-su… nation   day       2021-01-02
+ 3 us        smoothed_cli fb-su… nation   day       2021-01-03
+ 4 us        smoothed_cli fb-su… nation   day       2021-01-04
+ 5 us        smoothed_cli fb-su… nation   day       2021-01-05
+ 6 us        smoothed_cli fb-su… nation   day       2021-01-06
+ 7 us        smoothed_cli fb-su… nation   day       2021-01-07
+ 8 us        smoothed_cli fb-su… nation   day       2021-01-08
+ 9 us        smoothed_cli fb-su… nation   day       2021-01-09
+10 us        smoothed_cli fb-su… nation   day       2021-01-10
+# ℹ 141 more rows
+# ℹ 9 more variables: direction <dbl>, issue <date>,
+#   lag <dbl>, missing_value <dbl>, missing_stderr <dbl>,
+#   missing_sample_size <dbl>, value <dbl>, stderr <dbl>,
+#   sample_size <dbl>
+# ℹ Use `print(n = ...)` to see more rows
+```
+
+## Installation
+
+You can install the stable version of this package from CRAN:
+
+```
+install.packages("epidatr")
+pak::pkg_install("epidatr")
+renv::install("epidatr")
+```
+
+Or if you want the development version, install from GitHub:
+
+```
+# Install the dev version using `pak` or `remotes`
+pak::pkg_install("cmu-delphi/epidatr")
+remotes::install_github("cmu-delphi/epidatr")
+renv::install("cmu-delphi/epidatr")
```

Comment:
Are we switching the default epidatr branch to main?  Or keeping as dev because it's already on CRAN?  If the former, then we'll want an `@dev` here

Reply:
Probably safe to put @dev here, just in case we do switch. Will do.